adapters/rubicon/rubicon.go

```diff
@@ -1081,15 +1081,20 @@ func updateBidExtWithMetaNetworkId(bid rubiconBid, buyer int) json.RawMessage {
 	if bidExt != nil {
 		if bidExt.Prebid != nil {
 			if bidExt.Prebid.Meta != nil {
-				bidExt.Prebid.Meta.NetworkID = buyer
+				if buyer > 0 {
```

Comment:
Do you need these two ifs as these cases should be handled by the first line of this function?

Reply:
Line 1071 has the same checks, if you don't immediately return from failing those, then 1084 and 1087 will never fail, unless I am missing how these values might change somewhere in between.